csrankings-w.csv

```diff
@@ -453,7 +454,8 @@ Wissam Fawaz,Lebanese American University,http://soe.lau.edu.lb/ece/faculty-staf
 Witawas Srisa-an,University of Nebraska,http://cse.unl.edu/~witty,Nf_0d9MAAAAJ
 Witold Charatonik,University of Wroclaw,https://www.ii.uni.wroc.pl/~wch,mCyQ2RgAAAAJ
 Wlodek Drabent,IPI PAN,https://ipipan.waw.pl/instytut/pracownicy/wlodzimierz-drabent,bG-Ml1YAAAAJ
-Wlodek Zadrozny,UNC - Charlotte,https://cci.uncc.edu/directory/wlodek-zadrozny,PwdKImsAAAAJ
+Wlodek W. Zadrozny,UNC - Charlotte,https://sites.google.com/a/uncc.edu/wzadrozny/,PwdKImsAAAAJ

```

Comment:
Please remove this entry and ask DBLP to add an alias for it.

Reply:
I removed "Wlodek W. Zadrozny" and will ask dblp to create an entry for it.